sqlmesh/migrations/v0046_change_scd_string_to_column.py

```diff
@@ -0,0 +1,5 @@
+"""Changes the SCD Type 2 columns from strings to columns."""
```

Comment:
Please note that the `46` version already exists in main

Reply:
Thanks fixed 👍